nixos/modules/installer/cd-dvd/installation-cd-graphical-gnome.nix

```diff
@@ -10,10 +9,17 @@ with lib;
 
   services.xserver.desktopManager.gnome3.enable = true;
 
-  # Auto-login as root.
-  services.xserver.displayManager.gdm.autoLogin = {
+  # wayland can be problematic for some hardware
+  services.xserver.displayManager.defaultSession = "gnome-xorg";
+
+  services.xserver.displayManager.gdm = {
     enable = true;
-    user = "root";
+    # This might be problematic on a live system
```

Comment:
How?

Reply:
`Suspend the machine after inactivity.`, they could run into the issue with remote connections maybe?
It might be a problem for the installer media to have an incongruent configuration versus someone just using `gnome3.enable`. But I'm betting if we get the iso we can seed the generated configuration.nix from nixos-install with a comment about this option.